spacemacs/colors.vim

```diff
@@ -1,3 +1,4 @@
 " Pick color
+let g:WhichKeyDesc_Colors_ShowColorPicker = "<leader>Cl    ShowColorPicker"
```

Comment:
```suggestion
let g:WhichKeyDesc_Colors_ShowColorPicker = "<leader>Cl    color-list"
```

Reply:
Changed the name to something that corresponds to the keys and that follow the kebab-case convention.
What do you think?

Reply:
Et voila :)